reference/architecture/overview.md

```diff
@@ -104,7 +104,7 @@ url='https://github.com/containernetworking/cni/blob/master/SPEC.md#network-conf
 
 - **etcd**
 
-   `etcd` is a consistent, highly-available distributed key-value store that provides data storage for the {{site.prodname}} network, and forcommunications between components. `etcd` is supported for protecting only non-cluster hosts (as of {{site.prodname}} v3.1). For completeness, `etcd ` advantages are:
+   `etcd` is a consistent, highly-available distributed key-value store that provides data storage for the {{site.prodname}} network, and for communications between components. `etcd` is supported for protecting only non-cluster hosts (as of {{site.prodname}} v3.1). For completeness, `etcd ` advantages are:
```

Comment:
communication*?

Reply:
It could be either...communications meaning more than one type of communication between components, or singular. I don't feel strongly about it.